plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSinkProvider.java

```diff
@@ -99,14 +102,23 @@ public DeltaLakePageSinkProvider(
     public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorOutputTableHandle outputTableHandle, ConnectorPageSinkId pageSinkId)
     {
         DeltaLakeOutputTableHandle tableHandle = (DeltaLakeOutputTableHandle) outputTableHandle;
+        // TODO support ID and NAME column mapping
+        ColumnMappingMode columnMappingMode = NONE;
```

Comment:
shouldn't this take actual mapping + verify it's NONE?

Reply:
#17565 will add column mapping to DeltaLakeOutputTableHandle and handle the actual mapping here.